index.html

```diff
@@ -1540,6 +1540,86 @@ <h5>
           unnecessary and abusive requests.
         </p>
       </section>
+      <section>
+        <h3>
+          Fingerprinting and Data Leakage
+        </h3>
+        <h4>
+          Browser fingerprinting
+        </h4>
+        <p>
+          While the API ensures that no user data is ever shared without a
+          permission prompt, the longevity and uniqueness of real world
+          identifiers that are likely to be returned by the Digital Credentials
+          API make it a potential target for trackers and fingerprinters.
+        </p>
+        <p>
+          Even with selective disclosure, attackers might combine data from a
+          digital credential (such as the user's age, or the credential issuer,
+          timestamps, see [[[#leaking-incidental-data]]]) to reidentify and/or
+          fingerprint users.
+        </p>
+        <p>
+          This attack might be harder for third-party attackers (such as
+          scripts embedded on the verifier's pages, but not actively
+          collaborating with them for the purpose of tracking) because response
+          encryption is mandatory and responses should be decrypted on the
+          verifier's server. The verifier could thus ensure not to reflect back
+          decrypted information back to client-side JavaScript. Not all
+          verifiers will choose to do so, however.
```

Comment:
Asking websites to protect against fingerprinting would be a new approach, but not one likely to be effective. I expect that almost every verifier would reflect at least some of the data back to the site, in order to show the information collected, confirm it with the user, present some action or conclusion based on that data, etc. So it's not just that this protection might not work, but more that I don't think we expect that it will almost ever work.